exchanges/btse/btse.go

```diff
@@ -615,3 +615,24 @@ func (b *BTSE) calculateTradingFee(ctx context.Context, feeBuilder *exchange.Fee
 func parseOrderTime(timeStr string) (time.Time, error) {
 	return time.Parse(time.DateTime, timeStr)
 }
+
+// MillionPairs returns a map of symbol names which have a IsMillion equivalent
+func (m *MarketSummary) MillionPairs() map[string]struct{} {
```

Comment:
Can drop pointer receiver

Reply:
Can drop, yes, obviously 🙂 

However it's lighter to use the pointer here, right? ( By the 2 extra fields a slice by value contains )

What's the benefit of switching to pass-by-value here?